src/main/java/teammates/logic/api/LogsProcessor.java

```diff
@@ -85,7 +85,7 @@ public int getNumberOfLogsForEvent(Instant startTime, Instant endTime, LogEvent
             List<GeneralLogEntry> logs = logFetchResults.getLogEntries();
             total += logs.size();
             if (logFetchResults.getHasNextPage() && !logs.isEmpty()) {
-                logStartTime = logs.getLast().getTimestamp();
+                logStartTime = logs.get(logs.size()-1).getTimestamp();
```

Comment:
checkstyle wants a space around the "-"

size() - 1

Reply:
fixed it.